recipe/meta.yaml

```diff
@@ -42,7 +44,10 @@ about:
     effectiveness of tests. It can show which parts of your code are being
     exercised by tests, and which are not.
   doc_url: http://coverage.readthedocs.org
-  dev_url: https://bitbucket.org/ned/coveragepy
+  # the old dev_url is deprecated dev_url: https://bitbucket.org/ned/coveragepy
+  # the new dev_url was taken from read the docs
+  dev_url: https://github.com/nedbat/coveragepy
+  # the doc_source_url: seems to be deprecated
```

Comment:
The new `doc_src_url` should be located somewhere in the repository identified by `dev_url`.

Reply:
The `doc_source_url` was updated accordingly:

https://github.com/AnacondaRecipes/coverage-feedstock/blob/update_to_6.1.1/recipe/meta.yaml#L52